examples/Training/python/ROIs.py

```diff
@@ -79,21 +96,33 @@
 line.theZ = rint(theZ)
 line.theT = rint(theT)
 line.textValue = rstring("test-Line")
-roi.addShape(line)
+createROI(image, [line])
 
-# create a point shape and add to ROI
+# create an ROI with single point shape
 point = omero.model.PointI()
 point.cx = rdouble(x)
 point.cy = rdouble(y)
 point.theZ = rint(theZ)
 point.theT = rint(theT)
 point.textValue = rstring("test-Point")
-roi.addShape(point)
-
-
-# Save the ROI (saves any linked shapes too)
-r = updateService.saveAndReturnObject(roi)
-
+createROI(image, [point])
+
+
+def pointsToString(points):
+    """ Returns strange format supported by Insight """
+    points = ["%s,%s" % (p[0], p[1]) for p in points]
+    csv = ", ".join(points)
+    return "points[%s] points1[%s] points2[%s]" % (csv, csv, csv)
```

Comment:
@mtbc : I think this is the older format. @jburel / @dominikl : is the support for the newer available / carded?